homeassistant/components/technove/sensor.py

```diff
@@ -0,0 +1,161 @@
+        device_class=SensorDeviceClass.VOLTAGE,
+        state_class=SensorStateClass.MEASUREMENT,
+        entity_category=EntityCategory.DIAGNOSTIC,
+        value_fn=lambda station: station.info.voltage_out,
+    ),
+    TechnoVESensorEntityDescription(
+        key="max_current",
+        translation_key="max_current",
+        native_unit_of_measurement=UnitOfElectricCurrent.AMPERE,
+        device_class=SensorDeviceClass.CURRENT,
+        state_class=SensorStateClass.MEASUREMENT,
+        entity_category=EntityCategory.DIAGNOSTIC,
+        value_fn=lambda station: station.info.max_current,
+    ),
+    TechnoVESensorEntityDescription(
+        key="max_station_current",
+        translation_key="max_station_current",
+        native_unit_of_measurement=UnitOfElectricCurrent.AMPERE,
+        device_class=SensorDeviceClass.CURRENT,
+        state_class=SensorStateClass.MEASUREMENT,
+        entity_category=EntityCategory.DIAGNOSTIC,
+        value_fn=lambda station: station.info.max_station_current,
+    ),
+    TechnoVESensorEntityDescription(
+        key="current",
+        translation_key="current",
+        native_unit_of_measurement=UnitOfElectricCurrent.AMPERE,
+        device_class=SensorDeviceClass.CURRENT,
+        state_class=SensorStateClass.MEASUREMENT,
+        entity_category=EntityCategory.DIAGNOSTIC,
+        value_fn=lambda station: station.info.current,
+    ),
+    TechnoVESensorEntityDescription(
+        key="energy_total",
+        translation_key="energy_total",
+        native_unit_of_measurement=UnitOfEnergy.KILO_WATT_HOUR,
+        device_class=SensorDeviceClass.ENERGY,
+        state_class=SensorStateClass.TOTAL,
+        entity_category=EntityCategory.DIAGNOSTIC,
+        value_fn=lambda station: station.info.energy_total,
+    ),
+    TechnoVESensorEntityDescription(
+        key="energy_session",
+        translation_key="energy_session",
+        native_unit_of_measurement=UnitOfEnergy.KILO_WATT_HOUR,
+        device_class=SensorDeviceClass.ENERGY,
+        state_class=SensorStateClass.TOTAL_INCREASING,
+        entity_category=EntityCategory.DIAGNOSTIC,
+        value_fn=lambda station: station.info.energy_session,
+    ),
+    TechnoVESensorEntityDescription(
+        key="rssi",
+        translation_key="rssi",
+        native_unit_of_measurement=SIGNAL_STRENGTH_DECIBELS_MILLIWATT,
+        device_class=SensorDeviceClass.SIGNAL_STRENGTH,
+        state_class=SensorStateClass.MEASUREMENT,
+        entity_category=EntityCategory.DIAGNOSTIC,
+        entity_registry_enabled_default=False,
+        value_fn=lambda station: station.info.rssi,
+    ),
+    TechnoVESensorEntityDescription(
+        key="ssid",
+        translation_key="ssid",
+        icon="mdi:wifi",
+        entity_category=EntityCategory.DIAGNOSTIC,
+        entity_registry_enabled_default=False,
+        value_fn=lambda station: station.info.network_ssid,
+    ),
+    TechnoVESensorEntityDescription(
```

Comment:
Can this become an enum sensor? Which has all the possible options in the options field? Please don't use numbers as state but make it a snake case value

Reply:
I did the change but I'm not sure I did exactly what you asked, especially because of the part "Please don't use numbers as state but make it a snake case value". The values returned by the API are numbers. 

Is your recommendation that I should I take care of the conversion in the TechnoVE python library instead of here?

Reply:
Yes, that would be awesome when the library has a Enum for the status, because in most cases an user can't do much with an arbitrary number.

Reply:
I think it's best to make sure that the state of the enum sensor is a snake case string.

Enums are lovely!

Reply:
Thanks to both of you for the helpful feedback. I've implemented the status as an enum now.